client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -56,7 +59,8 @@ export default React.createClass( {
 			form: null,
 			isDialogVisible: false,
 			submissionCount: 0,
-			phoneCountryCode: 'US'
+			phoneCountryCode: 'US',
+			registrantExtraInfo: false
```

Comment:
Is this always boolean? If so, do you think it'd make it easier to understand if we prefix it with `has` or `needs`?